pkg/reconciler/v1alpha1/broker/broker_test.go

```diff
@@ -461,41 +461,6 @@ func TestReconcile(t *testing.T) {
 			},
```

Comment:
Please ignore broker_test.go. This file slipped in due to some other change I had in it